new Leetcode/38. Count and Say/38. Count and Say.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Created by sindhuhari on 1/24/18.
+ */
+/**
+ * @param {number} n
+ * @return {string}
+ */
+var countAndSay = function (n) {
+    if (n === 1) {
```

Comment:
this is again a special case. try not to do this. 

if you remove this special case, your program still works. check it.